showcase/src/app/shared/example-viewer/example-viewer.spec.ts

```diff
@@ -0,0 +1,92 @@
+import { HttpClientModule } from "@angular/common/http";
+import { BrowserAnimationsModule } from "@angular/platform-browser/animations";
+import { async, ComponentFixtureAutoDetect, ComponentFixture, TestBed } from "@angular/core/testing";
+import { MatButtonModule, MatIconModule, MatTabsModule, MatTooltipModule } from "@angular/material";
+import * as hljs from "highlight.js";
+import { HighlightJsModule, HIGHLIGHT_JS } from "angular-highlight-js";
+
+import { ExampleViewerComponent } from "./example-viewer.component";
+import { ExampleViewerService } from "./example-viewer.service";
+import { throwError, of } from "rxjs";
+
+export function highlightJsFactory(): any {
+	return hljs;
+}
+fdescribe("ExampleViewerComponent", () => {
```

Comment:
Don't forget to remove the "f" before the "describe", otherwise only this test will be executed in Travis!!